### DISCUSSION TITLE
Why does crl_download show that the CRL cannot be found?

### DISCUSSION THREAD

**Discussion by onepeople158** [openssl/openssl#25492] [Q&A] [answered]
Hello developer, I created a certificate with a CRL download address, but when I use the crl_download option, it shows that the CRL cannot be found (this address can successfully download the CRL).
![屏幕截图 2024-09-19 202842](https://github.com/user-attachments/assets/9484ea76-837b-4cf7-bcc7-a490866a0917)
![屏幕截图 2024-09-19 201321](https://github.com/user-attachments/assets/c0bfda39-4019-405f-8f29-c7bdca840d24)
![屏幕截图 2024-09-19 201232](https://github.com/user-attachments/assets/1943f95c-c29b-49fc-8605-1cde274d5fa2)

**Comment by t8m** [comment, marked as answer]
We do not support `https` URLs for CRL distribution points.

**Reply by mattcaswell** [reply]
Seems like we should add that feature!!!
